static/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -15,7 +15,7 @@ Once initialized in a project, DVC populates its installation directory
 - `.dvc/cache` - the cache directory will contain your data files. (The data
   directories of DVC repositories will only contain links to the data files in
   the cache, refer to
-  [Performance Optimization for Large Files](/docs/user-guide/cache-file-linking).)
+  [Cache File Linking](/docs/user-guide/cache-file-linking).)
```

Comment:
let's keep the previous value, the file name is not exactly right, but it's not of much importance to fix. We are iterating on the title value with @jorgeorpinel .

Reply:
Hi! Actually you may need to rebase from the upstream master branch because the path for this link has also changed. Please ping me if you need help on that @algomaster99 . Basically:
```console
git remote add upstream git@github.com:iterative/dvc.org.git # if you haven't
git fetch upstream
git rebase upstream/master
```
and hope for no conflicts 🤓  (otherwise follow the instructions in the `git` output).